docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+    dyanmo:
+      container_name: dynamo
+      image: amazon/dynamodb-local
+      ports: 
+        - 10005:8000
+      expose: 
+        - 8000
```

Comment:
IIUC `expose` is to allow container to container communication. Given we don't have another container, do we need to expose this port atm?

Reply:
This was based on the composer one and a brief look at the docs, it can probably go.